### PR TITLE
raftstore: add info log when check_and_compact

### PR DESCRIPTION
**Pull request by lerencao**
## What have you changed? (mandatory)

Add info log when compact_range finished.

## What are the type of the changes? (mandatory)

Improvement (non-breaking change which is an improvement to an existing feature)

## How has this PR been tested? (mandatory)

manual test

## Does this PR affect documentation (docs/docs-cn) update? (mandatory)

no

## Does this PR affect tidb-ansible update? (mandatory)

no

### CODE REVIEW

File: src/raftstore/store/worker/compact.rs [hunk 1]
```diff
@@ -158,6 +158,8 @@ impl Runnable<Task> for Runner {
                                 "compact range ({:?}, {:?}) for cf {:?} failed, error {:?}",
                                 start, end, cf, e
                             );
+                        } else {
+                            info!("compact range ({:?}, {:?}) for cf {:?} finished", start, end, cf);
```
Comment (AndreMouche) [inline src/raftstore/store/worker/compact.rs]: Will it print a lot of information during compaction?

Comment (lerencao) [reply src/raftstore/store/worker/compact.rs]: It equals to the times of manual compaction in RocksDB. In my observation, the common case is  a few hundreds or thousands per day.

Comment (siddontang) [reply src/raftstore/store/worker/compact.rs]: do you need to record the compact time here?

Comment (huachaohuang) [reply src/raftstore/store/worker/compact.rs]: How about moving the info log inside `compact_range_cf`, there is a timer inside that too, and we can also remove the info log in L136.

Comment (siddontang) [reply src/raftstore/store/worker/compact.rs]: great idea

/cc @lerencao

**Comment by lerencao** [comment]
updated.

File: src/raftstore/store/worker/compact.rs [hunk 2]
```diff
@@ -123,6 +125,13 @@ impl Runner {
             1, /* threads */
         );
         compact_range_timer.observe_duration();
```
Comment (overvenus) [inline src/raftstore/store/worker/compact.rs]: How about using `observe`?

```rust
let timer = Instant::now();
let duration = timer.elapsed();
COMPACT_RANGE_CF.with_label_values(&[&cf_name]).observe(duration_to_sec(duration));
```

Comment (siddontang) [reply src/raftstore/store/worker/compact.rs]: @overvenus seem we need to provide a function to hide `duration_to_sec` later, not let the user do.

**Pull request merged**